tests/integration/targets/rds_instance_snapshot/defaults/main.yml

```
@@ -0,0 +1,14 @@
+---
+# defaults file for rds_instance_snapshot
+
+# Create RDS instance
+instance_id: 'ansible-test-instance-{{ resource_prefix }}'
+username: 'testrdsusername'
+password: 'test-rds_password'
+db_instance_class: db.t3.micro
+allocated_storage: 10
+engine: 'mariadb'
+mariadb_engine_version: 10.3.31
```

(Non-blocking)

```suggestion
mariadb_engine_version: 10.3.31
```

We need to find a better way to handle this, Amazon drop support for old versions from time to time.  For example dropping python versions: https://github.com/ansible-collections/community.aws/pull/984

---

since our tests are not dependent on specific versions, we could increase them from time to time, like the boto3 version.    
imo it doesn't matter in this case, if we're going to deploy `mariadb_engine_version: 10.3.31` or `mariadb_engine_version: 10.5.13`.

---

@markuman is there anything blocking this from being merged?

---

imo no.